### ISSUE TITLE
Any examples for writing to devices?

### ISSUE THREAD

**Issue by bestplay** [kylelemons/gousb#29]
dev.OpenEndpoint()

got err:

`claim: libusb device or resource busy`

**Comment by bestplay** [comment]
I found a solution:

In gousb/device.go

**Call libusb_detach_kernel_driver before libusb_claim_interface**
```
        // Call libusb_detach_kernel_driver first
	if errno := C.libusb_detach_kernel_driver(d.handle, C.int(iface)); errno < 0 {
		fmt.Errorf("usb: libusb_detach_kernel_driver: %s", usbError(errno))
	}
        // Claim the interface
	if errno := C.libusb_claim_interface(d.handle, C.int(iface)); errno < 0 {
		return nil, fmt.Errorf("usb: claim: %s", usbError(errno))
	}
```

**Comment by zagrodzki** [comment]
That is correct. Pull #12 is open to do this automatically.

**Comment by zagrodzki** [comment]
Solved with #32
